.github/workflows/test.yml

```diff
@@ -30,6 +30,7 @@ jobs:
           poetry install
           protoc -I=$PROTOBUF_SRC_DIR --python_out=$PROTOBUF_SRC_DIR $PROTOBUF_SRC_DIR/*.proto
           2to3 --no-diff -n -w $PROTOBUF_SRC_DIR
+          find $PROTOBUF_SRC_DIR -type f -name "*_pb2.py" | xargs sed -i.bak -E 's/(USE_C_DESCRIPTORS == False:)/\1  # pragma: no cover/g'
```

Comment:
This works. A more generic way of doing it would be to generate a `.patch` with `git diff` and apply it. That helps catch the moment when this `find` + `sed` stop doing something useful.

Reply:
We could also just remove these 3 lines altogether since we've committed the generated (and properly fixed up) files, possibly adding a note in the developer guide what to do if editing or adding new `.proto` files.

Reply:
I captured this in https://github.com/opensearch-project/opensearch-sdk-py/issues/23#issuecomment-1733676710.